test/integration/features/basic.feature

```diff
@@ -84,6 +84,58 @@ Feature: Basic
       172.30.1.1:5000
       """
 
+  Scenario: Deploy ruby-ex s2i image
+    Given Minishift has state "Running"
+     When executing "oc login --username=developer --password=developer" retrying 24 times with wait period of 5 seconds
```

Comment:
Retrying 24 times with a wait time of 5 seconds? Why would you do that? If Minishift is running, the login needs to work directly. Why do you want to retry here? Did you see any problems? 

Retry is for cases where there is an expectation that the command can fail. This is not the case here.